core/bin/external_node/src/node_builder.rs

```diff
@@ -62,6 +62,8 @@ use crate::{
     Component,
 };
 
+const CRATE_VERSION: &str = env!("CARGO_PKG_VERSION");
```

Comment:
This is available as [`crate::metadata::server_version`](https://github.com/matter-labs/zksync-era/blob/f0b1416ca0f298e4bcd5fc615eaab963e3bc546f/core/bin/external_node/src/metadata.rs#L3).

Reply:
done